maps/randomvaults/objects.dm

```diff
@@ -76,3 +78,12 @@
 
 /obj/machinery/atmospherics/binary/msgs/rust_vault
 	name = "\improper Magnitno Priostanovleno Blok Khraneniya Gaza"
+
+/obj/item/weapon/paper/iou
+	name = "paper- 'IOU'"
+	info = "I owe you a rod of destruction. Redeemable at Milliway's at the end of time."
+
+/obj/item/weapon/gun/projectile/russian/empty/New()
+	..()
+	loaded.Cut()
```

Comment:
Firing a round keeps the fired round inside of it rather than adding a null spot.